### PR TITLE
DOC-4765: added Windows Known Issues to ##Limitations in README

### PR DESCRIPTION
**Pull request by jbondpdx**
Paging @cowofevil and @cyberious to review the added material for accuracy.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -73,6 +73,17 @@ An array of services to start, normally `puppet` and `mcollective`. If the array
 
 Supports only RPM-based distros: Redhat and Centos 5/6/7.
 
+###Known Issues
+
+The puppet_agent module has several known issues on Windows. This module supports Windows Server 2003 or later.
+
+* For Windows Server 2003, only x86 is supported.
+* Because the module directory has changed, modules installed manually on individual agents cannot be found after upgrading to Puppet 4.x. You should reinstall these modules with `puppet module install`.
```
Comment (MikaelSmith) [inline README.markdown]: This still isn't unique to Windows. Is there a reason we only call it out for Windows?

Comment (jbondpdx) [inline README.markdown]: I was only made aware of it as it relates to Windows. If this is a known issue on all platforms, I'll change it to a Known Issue apart from Windows.

Comment (cyberious) [inline README.markdown]: I think this is two seperate known issues. But only two in list so far

Comment (jbondpdx) [inline README.markdown]: Sorry @cyberious , I'm not sure what you mean. It's two separate issues that the modules can't be found? Could you clarify?

File: README.markdown [hunk 2]
```diff
@@ -73,6 +73,17 @@ An array of services to start, normally `puppet` and `mcollective`. If the array
...
+* To run `puppet agent -t interactive` on the agent, you must close the console and wait for the upgrade to finish.
```
Comment (cowofevil) [inline README.markdown]: I would reword to:

```
If you are performing the agent upgrade by executing `puppet agent -t' interactively in a console you must close the console and wait for the upgrade to finish before attempting to use the 'puppet' command again.
```

Comment (jbondpdx) [inline README.markdown]: Ah, that's helpful, thank you @cowofevil !

File: README.markdown (hunk 1 above)
Comment (jbondpdx) [inline README.markdown]: @MikaelSmith I'll move this out as a separate known issue. The only reason it's called out as Windows is because it was given to me in a list of "Known Issues with Windows". Thank you for clearing it up!

**Comment by cowofevil** [comment]
@jbondpdx Looks good to me, but you may need to rebase so I can merge.

File: README.markdown [hunk 3]
```diff
@@ -71,7 +71,25 @@ An array of services to start, normally `puppet` and `mcollective`. If the array
 
 ## Limitations
 
-Supports only RPM-based distros: Redhat and Centos 5/6/7.
+This module supports:
+
+* RHEL 5, 6, 7
+* Centos 5/6/7
```
Comment (MikaelSmith) [inline README.markdown]: Why '/' as opposed to ','?

Comment (jbondpdx) [inline README.markdown]: Oh, I just messed up. :-)

**Comment by jbondpdx** [comment]
Closing this PR, please see https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/42 instead.
